### PR TITLE
feat: load vehicle info default param

### PR DESCRIPTION
**Pull request by tkimura4**
## Related Issue(required)
https://github.com/autowarefoundation/autoware.universe/issues/1
https://github.com/autowarefoundation/autoware.universe/issues/123

## Description(required)

set default vehicle_info parameter to the launch file of the packages which use vehicle_info_util

package list
- control_performance_analysis
- lane_departure_checker
- obstacle_collision_checker
- vehicle_cmd_gate
- ground_segmentation
- behavior_path_planner
- behavior_velocity_planner
- freespace_planner
- obstacle_stop_planner
- surround_obstacle_checker
- simple_planning_simulator
- pacmod_interface
- scan_ground_filter
- ~trajectory_follower_nodes~ ( no launch file)

## Review Procedure(required)
launch packages as `ros2 launch simple_planning_simulator simple_planning_simulator.launch.py `, and check that following error message does not output.

> Failed to get parameter `wheel_radius`, please set it when you launch the node.

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by 1222-takeshi** [comment]
I couldn't get the build to pass, so I added this to the package.xml of ground_segmentation.
``` sh
  <depend>vehicle_info_util</depend>
```
Then, got the following output. :confused: 
image

I'm continuing to check it out.

File: simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py [hunk 1]
```diff
@@ -35,6 +35,16 @@ def generate_launch_description():
         description='Path to config file for vehicle characteristics'
     )
 
+    default_vehicle_info_param = os.path.join(
+        get_package_share_directory('vehicle_info_util'),
+        '/config/vehicle_info.param.yaml')
```
Comment (1222-takeshi) [inline simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py:40]:
```suggestion
        'config/vehicle_info.param.yaml')
```

File: perception/ground_segmentation/launch/scan_ground_filter.launch.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 import launch
 from launch.actions import DeclareLaunchArgument
```
Comment (1222-takeshi) [inline perception/ground_segmentation/launch/scan_ground_filter.launch.py:18]:
```suggestion
from ament_index_python.packages import get_package_share_directory
from launch.actions import DeclareLaunchArgument
```

**Comment by 1222-takeshi** [comment]
@tkimura4 
Could you fix build error ?

**Comment by 1222-takeshi** [comment]
Can't get vehicle_info in ground_segmentation, but I don't know why... 
image

**Comment by tkimura4** [comment]
@1222-takeshi 
I don't know the cause either, but it seems to happen only when using component.
I modified it to read the parameters directly from yaml file using python-yaml, can you try again?

**Review by 1222-takeshi — APPROVED** [review]
LGTM

**Comment by 1222-takeshi** [comment]
@tkimura4 
Could you fix DCO error ?

**Pull request merged**